stacks-signer/src/signer.rs

```diff
@@ -982,12 +855,7 @@ impl Signer {
     }
 
     /// Process a dkg result by broadcasting a vote to the stacks node
-    fn process_dkg(
-        &mut self,
-        stacks_client: &StacksClient,
-        point: &Point,
-        current_reward_cycle: u64,
-    ) {
+    fn process_dkg(&mut self, stacks_client: &StacksClient, point: &Point) {
```

Comment:
Can you give `point` a better name?  I have no idea looking at this function what it might be.

Reply:
Will do